packages/@aws-cdk/aws-eks/test/integ.eks-cluster.defaults.expected.json

```diff
@@ -1051,7 +1051,7 @@
       "Type": "AWS::AutoScaling::LaunchConfiguration",
       "Properties": {
         "ImageId": {
-          "Ref": "SsmParameterValueawsserviceeksoptimizedami113amazonlinux2recommendedimageidC96584B6F00A464EAD1953AFF4B05118Parameter"
+          "Ref": "SsmParameterValueawsserviceeksoptimizedami114amazonlinux2recommendedimageidC96584B6F00A464EAD1953AFF4B05118Parameter"
```

Comment:
@eladb I kind of want to add a small shell script to help the next contributor bump the latest kube version, what do you think?

Reply:
I went ahead and did it, tell me if you want to rename the script